src/oc/storage/api/orgs.clj

```diff
@@ -141,7 +140,7 @@
                              authors (:authors org)
                              author-reps (map #(org-rep/render-author-for-collection org %) authors)
                              update-count (if user
-                                            (count (update-res/list-updates-by-author conn org-id user-id))
+                                            0 ;(count (update-res/list-updates-by-author conn org-id user-id))
                                             0)]
```

Comment:
This is old code that is counting how many `stakeholder-updates` there were and putting the count on the link. We used to use this info in the web UI. I'll remove it.